Debian/bin/lib/Cyrus/Docker/Command/build.pm

```diff
@@ -18,6 +18,9 @@ sub opt_spec {
   return (
     [ 'recompile|r', 'recompile, make check, and install a previous build' ],
     [ 'no-cunit|n',  'do not run make check' ],
+    [ 'jobs|j=i',    'specify number of parallel jobs (default: 8) to run for make/make check',
+                     { default => 8 },
+    ],
```

Comment:
Non-required suggestion:  give `dar` a config option to change the default.  That'll be a little fiddly, though.  So, just a thought for the future.

Reply:
For sure